src/components/@core/navigation-menu/light/left-menu/logo.tsx

```diff
@@ -23,15 +30,24 @@ const Logo = styled.div`
     }
   }
 
-  button {
+  button.ibpmenu {
     display: none;
-    padding: 0.5rem;
+    padding: 0.5rem 1rem;
     font-size: 1.5rem;
   }
 
   ${Mq.max.sm} {
     width: 100%;
-    button {
+    a {
+<<<<<<< HEAD
+=======
+      margin: 0px 5px;
+>>>>>>> acede83abff5722c5444f6754219707393919b20
```

Comment:
This is merge conflict !!

Please review your code before commit or push VSCode